### PR TITLE
[`Frontend`] : Port vercel_hover

### PR DESCRIPTION
**Pull request by baseplate-admin**

### CODE REVIEW

File: backend/django_core/static_src/components/minor/SideBar/Index.svelte
Comment (tokitouq) [inline backend/django_core/static_src/components/minor/SideBar/Index.svelte:1]: I think we should seperate Vercel logic and Sidebar component.
cause we use this effect on other parts as well.

**Review by tokitouq — COMMENTED** [review]
Consider this

File: backend/django_core/static_src/components/minor/SideBar/Index.svelte
Comment (baseplate-admin) [reply backend/django_core/static_src/components/minor/SideBar/Index.svelte:1]: Maybe? Do we need that?

Comment (tokitouq) [reply backend/django_core/static_src/components/minor/SideBar/Index.svelte:1]: > Do we need that?

Yea? we added this effect on avatar dropdown, it looks good with this NGL.

File: backend/django_core/static_src/components/svelte/VercelHover.svelte [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                is_hovered_from_prev_el = false;
+            }, GLIDER_TRANSITION_DURATION);
+        };
+</script>
+
+<glider-container
+    bind:this={glider_container_element}
+    class={glider_container_class}
+>
+    <active-glider
```
Comment (tokitouq) [inline backend/django_core/static_src/components/svelte/VercelHover.svelte:68]: rename this custom elements?

File: backend/django_core/templates/anime/_layout.html [hunk 2]
```diff
@@ -101,7 +101,7 @@
                             </div>
                             <coreproject-specific-search-modal></coreproject-specific-search-modal>
                            
-                                <coreproject-sidebar glider_container_class='mt-[2.8125vw] flex flex-col items-center gap-[0.75vw]'  active_element_class='rounded-[0.75vw] bg-white/10'  direction='vertical' duration=200 />
+                                <coreproject-sidebar><?coreproject-sidebar>
```
Comment (tokitouq) [inline backend/django_core/templates/anime/_layout.html]: TYpo here `?`

Comment (baseplate-admin) [reply backend/django_core/templates/anime/_layout.html]: Good catch

**Pull request merged**